dvc/command/version.py

```diff
@@ -91,6 +91,8 @@ def get_linktype_support_info(repo):
 
         fname = "." + str(uuid.uuid4())
         src = os.path.join(repo.cache.local.cache_dir, fname)
+        if not os.path.exists(repo.cache.local.cache_dir):
+            repo.cache.local.makedirs(repo.cache.local.path_info)
```

Comment:
@efiop , why are we doing this on `version`?

Reply:
Because `.dvc/cache` might not exist, if no files have been added to it yet. I'll add the note, thanks!

Reply:
Also it might cause the same issue, when `dvc version` is ran before `dvc config cache.shared group`, so we shouldn't create it here, but rather just skip this check.